web_src/js/index.js

```diff
@@ -728,17 +728,8 @@ async function initRepository() {
 
   // Options
   if ($('.repository.settings.options').length > 0) {
-    $('#repo_name').keyup(function () {
```

Comment:
@silverwind can we just delete this?

Reply:
Yes, it's dead code:

Ref: https://github.com/gogs/gogs/search?q=repo-name-change-prompt
Ref: https://github.com/go-gitea/gitea/search?q=repo-name-change-prompt

Reply:
I did not check history why this prompt was removed, but feel free to dig into it :)

Reply:
🔪 it